python/paddle/nn/layer/conv.py

```diff
@@ -32,13 +52,13 @@
 __all__ = []
 
 
-def _get_default_param_initializer(num_channels, filter_size):
+def _get_default_param_initializer(num_channels: int, filter_size: int) -> paddle.Tensor:
     filter_elem_num = num_channels * np.prod(filter_size)
     std = (2.0 / filter_elem_num) ** 0.5
     return Normal(0.0, std)
 
 
-def _reverse_repeat_list(t, n):
+def _reverse_repeat_list(t: Sequence[int | str], n: int) -> list:
```

Comment:
可以尝试

``` python
_T = TypeVar("_T")

def _reverse_repeat_list(t: Sequence[_T], n:int) -> list[_T]:
    """Reverse the order of `t` and repeat each element for `n` times.
    This can be used to translate padding arg used by Conv and Pooling modules
    to the ones used by `F.pad`.
    """
    return [x for x in reversed(t) for _ in range(n)]

```